app/views/idv/verify_info/show.html.erb

```diff
@@ -15,6 +15,24 @@ locals:
   <!-- Needed by form steps wait javascript -->
 </div>
 
+<% if @had_barcode_read_failure %>
```

Comment:
is this something that used to be built in to the FSM?

Reply:
Yes it was part of the Verify template and calls the RedoDocumentCaptureAction which is still an FSM step. Added [LG-8830](https://cm-jira.usa.gov/browse/LG-8830) for next steps.

This was already merged to main as part of #7748 so it could be taken out of this PR or handled during rebase/merge with main.

Reply:
Yeah, this change was merged in a different PR. I just merged main in so it shouldn't be here anymore.